Gemfile.lock

```diff
@@ -283,9 +286,11 @@ DEPENDENCIES
   rubocop (~> 0.36.0)
   ruby-saml (~> 1.3.0)
   sdoc (~> 0.4.0)
+  shoulda-matchers
```

Comment:
can we not go down this route? shoulda-matchers are bad testing practice imo

we can use acceptance tests and unit tests to test validation, using shoulda-matchers is a bad and fragile way to get the same or similar testing done.